lib/ts/Arena.h

```diff
@@ -21,12 +21,13 @@
   limitations under the License.
  */
 
-#ifndef __ARENA_H__
-#define __ARENA_H__
+#pragma once
```

Comment:
That policy isn't what I heard.   What I heard is "as you touch it, upgrade to current policy / practice but otherwise leave it alone."